### ISSUE TITLE
Convert crown images to buttons

### ISSUE THREAD

**Issue by JCook21** [ProcessMaker/modeler#839]
img elements have an implicit role of "img" and so this shouldn't be set. See https://www.w3.org/TR/html-aria/#document-conformance-requirements-for-use-of-aria-attributes-in-html.

The best solution for this should be to use a button element with the `img` as its child. The button can then have an `aria-label`, and the img's alt text can be set to an empty string `""` and have a `role` of "presentation". The button would of course need to be slighly re-styled. This would make more the most accessible and semantically correct structure, in my opinion.

_Originally posted by @chipit24 in https://github.com/ProcessMaker/modeler/pull/838_
